tasks/sign-index-image/README.md

```diff
@@ -29,5 +29,8 @@ data:
 ## Changes in 1.1.0
 - change the task to use the `internal-request` script
 
+## Changes since 0.2.0
```

Comment:
there is something messed up here with the rebase. We shouldn't be adding a changelog entry that isn't the latest. Maybe your PR should now be adding 1.2.0 and an entry for Changes in 1.2.0?